spec/index.html

```diff
@@ -599,6 +609,17 @@ <h3>RDF Terms</h3>
       nor to a blank node with the <a>blank node identifier</a>
       <code>http://example.org/</code>.</p>
 
+    <p><dfn>RDF term equality</dfn>:
+      Two [=RDF terms=] |t| and <var>t'</var> are considered equal if and only if
+      one of the following four conditions holds:</p>
+
+    <ul>
+      <li>|t| and <var>t'</var> are [=IRIs=] that are [=IRI equality|equal=].</li>
+      <li>|t| and <var>t'</var> are [=literals=] that are [=literal term equality|equal=].</li>
+      <li>|t| and <var>t'</var> are [=blank nodes=] that are [=blank node equality|equal=].</li>
+      <li>|t| and <var>t'</var> are [=triple terms=] that are [=triple term equality|equal=].</li>
```

Comment:
```suggestion
      <li>|t| and <var>t'</var> are [=IRIs=] that are [=IRI equality|equal=] (per [=IRI equality=]).</li>
      <li>|t| and <var>t'</var> are [=literals=] that are [=literal term equality|equal=] (per [=literal term equality=]).</li>
      <li>|t| and <var>t'</var> are [=blank nodes=] that are [=blank node equality|equal=] (per [=blank node equality=]).</li>
      <li>|t| and <var>t'</var> are [=triple terms=] that are [=triple equality|equal=] (per [=triple equality=]).</li>
```

I would not fight for this change, but I think it makes the rendered text more readable. It can easily be overlooked that the four occurrences of "equal" do not refer to the same definition.

Also, I think it is better to refer directly to "triple equality" than to "triple term equality". See my remark below.

Reply:
Or the pattern:
```
  <li>|t| and <var>t'</var> are [=IRIs=] that are [=IRI equality|equal as IRIs=].</li>
```